QuizApp/App/Screens/Quiz/CategoryCell.swift

```diff
@@ -22,9 +22,8 @@ final class CategoryCell: UICollectionViewCell {
 
 extension CategoryCell {
 
-    func set(for category: Category) {
-        titleLabel.text = category.type.rawValue
-        titleLabel.textColor = category.color
+    func set(for quiz: QuizModel) {
```

Comment:
`QuizModel` is a use case layer model. You should have a view model for the UI layer.